program-runtime/src/invoke_context.rs

```diff
@@ -719,19 +720,28 @@ impl<'a> InvokeContext<'a> {
         };
 
         for entry in self.builtin_programs.vec.iter() {
-            if entry.program_id == builtin_id {
+            if entry.0 == builtin_id {
+                const ENTRYPOINT_KEY: u32 = 0x71E3CF81;
+                let process_instruction = match &entry.1.program {
+                    LoadedProgramType::Builtin(_name, program) => program
+                        .lookup_function(ENTRYPOINT_KEY)
```

Comment:
Where is `lookup_function` defined?

Reply:
https://github.com/solana-labs/rbpf/blob/e9c2d206e18a5e759b63f39353b029dec6dc4247/src/vm.rs#L141